widgets/AttributesTable/Table.js

```diff
@@ -302,6 +302,15 @@ define([
             this.gridOptions.sort = [];
         },
 
+        clearSelectedRecords: function () {
```

Comment:
Should method be renamed to `clearSelected` or similar since it included graphics and isn't specific to just dgrid rows or dstore records?